### PR TITLE
Update 3.0 docs accordingly to min PHP version requirement

### PR DESCRIPTION
**Pull request by ogizanagi**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 3.0+ |
| Fixed tickets | #5947 |

I'm not sure about the politic regarding the removal of anything mentioning earlier version of PHP for the 3.0 docs ? Should it be reverted ?
Regarding this PR: #5585, it's seems coherent to me to do the same for PHP versions.

### CODE REVIEW

File: create_framework/introduction.rst [hunk 1]
```diff
@@ -109,9 +109,8 @@ start with the simplest web application we can think of in PHP::
 
     printf('Hello %s', $input);
 
-If you have PHP 5.4, you can use the PHP built-in server to test this great
-application in a browser (``http://localhost:4321/index.php?name=Fabien``).
-Otherwise, use your own server (Apache, Nginx, etc.):
+You can use the PHP built-in server to test this great
+application in a browser (``http://localhost:4321/index.php?name=Fabien``):
```
Comment (xabbuh) [inline create_framework/introduction.rst]: We should fix the position of the line break now (move a few words on the previous line).

**Comment by xabbuh** [comment]
This looks good to me. :+1:

File: components/dom_crawler.rst [hunk 2]
```diff
@@ -300,12 +300,6 @@ and :phpclass:`DOMNode` objects:
 
     The ``html`` method is new in Symfony 2.3.
 
-    .. caution::
-
-        Due to an issue in PHP, the ``html()`` method returns wrongly decoded HTML
-        entities in PHP versions lower than 5.3.6 (for example, it returns ``•``
-        instead of ``&bull;``).
```
Comment (WouterJ) [inline components/dom_crawler.rst]: this should actually also make it into 2.7 as it requires 5.3.9.

**Comment by WouterJ** [comment]
Thank you for finding all notes about old PHP versions and removing them @ogizanagi!

I've fixed the 2 comments of @xabbuh and myself with e6ead2b and 8c17033

**Comment by felipsmartins** [comment]
Nice work, guys!

**Pull request merged**